### DISCUSSION TITLE
how to change the light color of each lamp in minecraft to the default color

### DISCUSSION THREAD

**Discussion by SkySe7eN1997** [jebbyk/OSBES-minecraft-bedrock-edition-shader#242] [Q&A] [answered]
how to change the light color of each lamp in minecraft to the default color

**Comment by hope3x** [comment, marked as answer]
It's in **shaders/glsl/includes/options/preset/lighting.glsl**

Change the value of `POINT_LIGHTS_TINT`

**Reply by josiahblaisdell** [reply]
Hi, when I change this value it does not change the color of my lamps, they are still the default color.

**Reply by hope3x** [reply]
> Hi, when I change this value it does not change the color of my lamps, they are still the default color.

Did you restart the game?
